### DISCUSSION TITLE
Naming

### DISCUSSION THREAD

**Discussion by felhasznalonev** [BoidCMS/BoidCMS#24] [Q&A] [answered]
What does word "Boid" mean?

**Comment by sysadev** [comment, marked as answer]
"BoidCMS" may not be a word you've heard before, and that's because it doesn't represent anything in particular. The name was actually inspired by a concept in artificial life called "Boids, (https://en.m.wikipedia.org/wiki/Boids)" which refers to the behavior of flocks of birds. Despite its unusual name, BoidCMS was built to simplify the process of creating simple websites and blogs, with a focus on extensibility and ease of use.
